lms/static/sass/views/_login-register.scss

```diff
@@ -67,7 +67,7 @@ $sm-btn-linkedin: #0077b5;
         }
```

Comment:
General: Please make sure your styling rules follow our [property order guidelines](https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#property-and-declaration-order).